integration/spark/src/main/common/java/io/openlineage/spark/agent/lifecycle/RddExecutionContext.java

```diff
@@ -67,7 +68,7 @@
   private List<URI> outputs;
   private String jobSuffix;
 
-  public RddExecutionContext(int jobId, EventEmitter sparkContext) {
+  public RddExecutionContext(OpenLineageContext context, int jobId, EventEmitter sparkContext) {
```

Comment:
_Minor_: Looks like `sparkContext` should be renamed to `eventEmitter`?

Reply:
I've done this in one of the other PRs- I'd rather not go through the hassle of fixing the conflicts here :)